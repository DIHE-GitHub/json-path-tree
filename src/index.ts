export { getPathTree } from "./utils/json-path-tree"