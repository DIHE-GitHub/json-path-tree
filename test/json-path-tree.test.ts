import { getPathTree } from "../src";

const mockJson = {
  a: { b: [{ c: 3 }] }
}

test('package main method simeple snapshot test', () => {
  const tree = getPathTree(mockJson)
  expect(tree).toMatchSnapshot();
});