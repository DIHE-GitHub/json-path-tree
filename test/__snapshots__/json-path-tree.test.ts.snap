// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`package main method simeple snapshot test 1`] = `
{
  "a": {
    "b": {
      "c": null,
    },
  },
}
`;
